src/api/status/public/js/post/index.js

```diff
@@ -0,0 +1,18 @@
+const apiHost = new URL(location).origin;
+
+const fetchPostsCount = () =>
+  fetch(`${apiHost}/v1/posts`, { method: 'HEAD' }).then((res) => res.headers.get('x-total-count'));
+
+const getPostsCount = async () => {
+  try {
+    const total = await fetchPostsCount();
```

Comment:
I think it's better to change "total" to "numberOfPosts" which makes it much more readable

Reply:
Makes sense. I will change it